### PR TITLE
[New Rule] Security Software Discovery using WMIC

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/386

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
**Alert Example:**

image

File: rules/windows/discovery_security_software_wmic.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/10/19"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/19"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies use of WMIC to discover certain System Security Settings such as AntiVirus or Host Firewall details."
```
Comment (threat-punter) [inline rules/windows/discovery_security_software_wmic.toml:9]:
```suggestion
description = """
Identifies the use of Windows Management Instrumentation Command (WMIC) to discover certain System Security Settings
such as AntiVirus or Host Firewall details.
"""
```

File: rules/windows/discovery_security_software_wmic.toml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Security Software Discovery using WMIC"
+risk_score = 21
```
Comment (threat-punter) [inline rules/windows/discovery_security_software_wmic.toml:15]:
```suggestion
risk_score = 47
```

**Review by threat-punter — APPROVED** [review]
👍

**Review by peasead — APPROVED** [review]
LGTM

**Pull request merged**